bundle/manifests/falcon-operator.clusterserviceversion.yaml

```diff
@@ -69,11 +69,11 @@ spec:
       name: falconnodesensors.falcon.crowdstrike.com
       version: v1alpha1
   description: |-
-    Falcon Operator installs CrowdStrike Falcon Container Sensor or CrowdStrike Falcon Node Sensor on the cluster.
-    Falcon Operator is an open source project, not a CrowdStrike product. As such it carries no formal support, expressed or implied.
+    The CrowdStrike Falcon Operator installs CrowdStrike Falcon Container Sensor or CrowdStrike Falcon Node Sensor on the cluster.
+    The CrowdStrike Falcon Operator is an open source project and not a CrowdStrike product. As such, it carries no formal support, expressed, or implied.
```

Comment:
Does it make sense to make this exact same amends to the README.md. It contains the same language.

Reply:
Totally. Updated the readme.